common/build/eslint-config-fluid/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fluidframework/eslint-config-fluid",
-	"version": "4.0.0",
+	"version": "5.0.0",
```

Comment:
Note

Adding a new custom rule is a breaking change.

Reply:
What are the rules for what qualifies as breaking for linting config? I know the removal of minimal was breaking, but I would have guessed this change was minor.... even if it takes work to pull in the update.

Reply:
With the `no-member-release-tags` rule from the `@fluid-internal/eslint-plugin-fluid` package now consumed in this package, (imported as a plugin in `minimal-deprecated.js` and also extended in `recommended.js`), the new `eslint-config-fluid` will now raise linter errors and the local builds will fail.

I thought this consist as a breaking change thus bumping to `5.0.0` is indeed the right choice. 

Looping @alexvy86

Reply:
I don't think it hurts to go major. It just seemed like there would be a lot of majors if any change that could produce a new error was considered breaking. Mostly I wanted to understand the real rules. It is a little hard to check the past even though I made a weak attempt.

Reply:
The rationale for me is that anyone consuming it (should be no one but us, but still) could see new errors when linting the same codebase where they see no errors with v4.x. AFAIK minors in semver shouldn't require any work to consume (except if a bug is being fixed, maybe)?

Reply:
> It just seemed like there would be a lot of majors if any change that could produce a new error was considered breaking.

That's true. I didn't look at how other eslint plugins handle their versions. I wouldn't mind making this a minor update, except for the fact that it is (unfortunately, IMO) in the @fluidframework scope which we've defined as consumable by third parties, so right now would err on the side of major.

Reply:
> > It just seemed like there would be a lot of majors if any change that could produce a new error was considered breaking.
> 
> That's true. I didn't look at how other eslint plugins handle their versions. I wouldn't mind making this a minor update, except for the fact that it is (unfortunately, IMO) in the @fluidframework scope which we've defined as consumable by third parties, so right now would err on the side of major.

Well, we are changing the scope of packages right now. This package has rules assuming internal use. That is going to need straightened out. I don't know the intents of the plug-in.

cc @Josmithr, @ChumpChief, @DLehenbauer

Reply:
Also true! And I'd be happy to see this move to @fluid-internal, and since it's an independent package I guess it's easy-ish to do at any time.